CONTRIBUTING.md

```diff
@@ -21,9 +21,44 @@ In OpenCADC, the publicly released code lives in the main (or master) branch of
 
 ### Java Development Guidelines
 
-TODO
-* checkstyle
-* CI
+Java code is licenced under the Affero GPL version 3 (AGPL-3.0) and each file originally created by CADC staff includes a standard bilingual copyright notice with `(C) Government of Canada ...`. Files originally created and contributed should be under the same license and may/should/usually contain their own suitable `(C)` claim; copyright is assumed to be assigned (to GoC) if the standard bilingual notice if included. **TBD**
+
```

Comment:
Perhaps this licence info applies to all opencadc software and could be promoted out of this section?
There is also the LICENCE file that could be referenced here.
I don't think we need to be overly specific.
Should we remove the TBD?

Reply:
Could be promoted but needs to be somewhere. I don't know if the convention to put this in every file is the same in python, notebooks, or other kinds of "source" so only wanted to address it in java.

Also, we do need some stance on copyright, which I was saying is "assigned to GoC" if it has  the std header and if someone contributes a new file they can retain (C).... multiple copyright holders is always a bit of a pain and we don't want multiple at the individual file level. So a bug fix always assigns copyright and the other extreme would be a new module in the repo (eg tap.git/cadc-tap-server-bigquery) where it's easy enough to have a completely different copyright holder (I would expect it).
New files in tap.git/cadc-tap-server? I'd prefer copyright assignment to GoC but will need a real life case to figure it out.

Reply:
clarified but retained for now